data/yam/autoyast/raid_encrypted_home.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0"?>
```

Comment:
I think this copied from data/yam/autoyast/raid_encrypted_home_x86_64.xml and updated for addon section, now works for x86_64, not sure whether this works for other arches( in the future may extend this test to all arches). For now I think we can update the data/yam/autoyast/raid_encrypted_home_x86_64.xml, we can create a new file for 15SP7 if proved that different from <=15SP6.

Reply:
After a checking on data/yam/autoyast/raid_encrypted_home_x86_64.xml, I found that this yaml file is now used on YaST MU job, do you think it is fine to update this file? And usually the yaml files used for MU jobs, it would add MU repos, I think we can add a new one file for YaST job group case now. WDYT?

Reply:
Please update the profile follow the yam/autoyast/support_images/create_hdd_maintenance.xml.ep, then try to verify it in maintenance test and yast group, thanks.

Reply:
updated.